fastlane/lib/fastlane/actions/install_provisioning_profile.rb

```diff
@@ -0,0 +1,58 @@
+require 'shellwords'
+
+module Fastlane
+  module Actions
+    class InstallProvisioningProfileAction < Action
+      def self.run(params)
+        absolute_path = File.expand_path(params[:provisioning_profile_path])
+        FastlaneCore::ProvisioningProfile.install(absolute_path)
+      end
+
+      def self.description
+        "Install provisioning profile from inputfile"
+      end
+
+      def self.details
+        "Install provisioning profile from inputfile for current user"
+      end
+
+      def self.authors
+        ["SofteqDG"]
+      end
+
+      def self.category
+        :code_signing
+      end
+
+      def self.is_supported?(platform)
+        [:ios, :mac].include?(platform)
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :provisioning_profile_path,
+                                  env_name: "FL_INSTALL_PROVISIONING_PROFILE_PATH",
+                               description: "Path to provisioning profile",
+                                  optional: false,
+                                      type: String,
+                              verify_block: proc do |value|
```

Comment:
What if the file path contains the spaces? Shouldn't we `shellescape` it? 🤔 

Adding a unit test for it would be great 👍

Reply:
@mollyIV No need for a `shellescape` here! This path is only used in Ruby stuff so it doesn't actually get pushed down to a shell command anywhere on our end 🙃